src/verify.c

```diff
@@ -805,6 +809,38 @@ int verify_main(int argc, char **argv)
 	}
 	grabtime_stop(&times);
 
+	/* When installing, check disk space availability */
+	/* This assumes there's no pre-installed/partially installed bundles in path_prefix */
+	if (cmdline_option_install) {
+		struct list *ptr = list_head(official_manifest->submanifests);
```

Comment:
This block of code is almost identical to the code in `src/bundle.c`. Looks like it could be its own function.